### PR TITLE
pgvector: Use SA's `bulk_save_objects` method for inserting embeddings

### PR DESCRIPTION
**Pull request by amotl**
## About

Using SQLAlchemy's bulk insert methods can substantially increase performance.

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/cratedb/base.py [hunk 1]
```diff
@@ -168,9 +168,11 @@ def add_embeddings(
             self.delete_collection()
         self.create_tables_if_not_exists()
         self.create_collection()
-        return super().add_embeddings(
+        embedding_ids = super().add_embeddings(
             texts=texts, embeddings=embeddings, metadatas=metadatas, ids=ids, **kwargs
         )
+        refresh_table(self.Session(), self.EmbeddingStore)
```
Comment (matriv) [inline libs/langchain/langchain/vectorstores/cratedb/base.py:174]: This is irrelevant to the bulk, it was missing anyway, right?

**Review by matriv — APPROVED** [review]
LGTM, left a question.

File: libs/langchain/langchain/vectorstores/cratedb/base.py (hunk 1 above)
Comment (amotl) [reply libs/langchain/langchain/vectorstores/cratedb/base.py:174]: Thanks for asking. Yes, this synchronization operation was missing anyway, but it made the flaw more appearant: Making the insert operation faster, made the corresponding tests trip right away.

Comment (matriv) [reply libs/langchain/langchain/vectorstores/cratedb/base.py:174]: thx!

**Pull request merged**